### PR TITLE
updated FAQs and Deployment script

### PR DESCRIPTION
**Pull request by pranchalsomani**

### CODE REVIEW

File: FAQs/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
  2. [How to add new subscriptions or mangement groups after deploying AzTS?](#2-how-to-add-new-subscriptions-or-management-groups-after-deploying-azts)
  3. [On running the AzTS installation command (`Install-AzSKTenantSecuritySolution`) I am getting an error message "Tenant ID, application ID, principal ID, and scope are not allowed to be updated."](#3-on-running-the-azts-installation-command-install-azsktenantsecuritysolution-i-am-getting-an-error-message-tenant-id-application-id-principal-id-and-scope-are-not-allowed-to-be-updated)
  4. [While installing AzTS solution I have provided my preferences for telemetry collection i.e. anonymous AzTS usage data and organization/team contact details. How do I update my preferences now?](#4-while-installing-azts-solution-i-have-provided-my-preferences-for-telemetry-collection-ie-anonymous-azts-usage-data-and-organizationteam-contact-details-how-do-i-update-my-preferences-now)
+ 5. [On running the Autoupdater function getting some error related to function runtime version.](#5-on-running-the-autoupdater-function-getting-some-error-related-to-function-runtime-version)
```
Comment (ZiaAhmad-PM) [inline FAQs/README.md:13]: Could we pls. specify what error as "some error" is too ambiguous as a FAQ topic. Pls. take reference from point no. 3. 

Take for example: 
**On running the Autoupdater function (XYZ..) I am encountering an error related to conflicts in function runtime version**

File: FAQs/README.md [hunk 2]
```diff
@@ -70,6 +71,28 @@ To update the telemetry preferences, go to resource group where AzTS solution ha
   - If AIConfigurations\_\_AnonymousUsageTelemetry\_\_LogLevel is either 'All' or 'Onboarding', then specify the team's contact DL in this.
   - If AIConfigurations\_\_AnonymousUsageTelemetry\_\_LogLevel is either 'Anonymous' or 'None', then specify 'N/A' in this.
   
+### **5. On running the Autoupdater function getting some error related to function runtime version.**
+
+
+- Run below command for AzSK-AzTS-AutoUpdater-xxxxx function app.
```
Comment (ZiaAhmad-PM) [inline FAQs/README.md:77]: Call these as Step 1, Step 2 and Step 3

File: FAQs/README.md [hunk 3]
```diff
@@ -70,6 +71,28 @@ To update the telemetry preferences, go to resource group where AzTS solution ha
...
+  Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+  ```
+- Run below command for AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx.
+
+  ``` Powershell
+  Update-AzFunctionAppSetting -AppSetting @{"FUNCTIONS_EXTENSION_VERSION" = "~4"} -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME> -Force
+
+  Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+  ```
+- Try running the Autoupdater once again.
```
Comment (ZiaAhmad-PM) [inline FAQs/README.md:95]: We need not start with "Try running..."

Step 3: Re-rerun the Autoupdater after completing Step 2 and the operation should complete successfully. This also ensures that the ..... (whatever the function run time was supposed to get updated to).

**Review by ZiaAhmad-PM — CHANGES_REQUESTED** [review]

File: FAQs/README.md [hunk 4]
```diff
@@ -70,9 +71,34 @@ To update the telemetry preferences, go to resource group where AzTS solution ha
...
+- Step-2 : Run below command for AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx.
+
+    ``` Powershell
+    # Update function runtime. 
+    Update-AzFunctionAppSetting -AppSetting @{"FUNCTIONS_EXTENSION_VERSION" = "~4"} -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME> -Force
+
+    # Update Net framework version
+    Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+    ```
+- Step-3. Re-run the AutoUpdater function after completing Step 2 and the operation should complete successfully.This also ensures that the function runtime is updated to V4 for AzSK-AzTS-AutoUpdater-xxxxx, AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx function app.
```
Comment (ZiaAhmad-PM) [inline FAQs/README.md:99]: - Step-3. Re-run the AutoUpdater function after completing Step 2 and the operation should complete successfully. This also ensures that the function runtime is updated to V4 for AzSK-AzTS-AutoUpdater-xxxxx, AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx function app.

**Review by ZiaAhmad-PM — CHANGES_REQUESTED** [review]

**Review by ZiaAhmad-PM — APPROVED** [review]

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**